manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -0,0 +1,66 @@
+# Installed for each revision - not installed for cluster resources ( cluster roles, bindings, crds)
```

Comment:
If we make the suggestions above this is just a single object. What benefit do we get making a complete helm chart rather than just a yaml file under samples/?